config/environments/development.rb

```diff
@@ -51,4 +51,12 @@
   # Use an evented file watcher to asynchronously detect changes in source code,
   # routes, locales, etc. This feature depends on the listen gem.
   config.file_watcher = ActiveSupport::EventedFileUpdateChecker
+  config.action_mailer.smtp_settings = {
+   :address              => "smtp.gmail.com",
+   :port                 => 587,
```

Comment:
I would like to review this with you more in person tomorrow. There are some very interesting pieces here.